api.md

```diff
@@ -25,6 +25,24 @@
 > - [show_tablespaces](#show_tablespaces)
 > - [time_bucket](#time_bucket)
 
+## TimescaleDB informational views [](timescaledb_information)
```

Comment:
Is this meant to be a top level section (i.e. in the sidebar)?

Reply:
i wanted feedback on this so i didnt add any sections but diana wanted this to go under utility which is where i put it now